### PR TITLE
Updated Master to Main

### LINKED ISSUE #570: Renaming `master` to `main` has lead to link rot
🐞 **Problem**

We recently renamed the `master` branch to `main` and it has lead to link rot. While GitHub automatically interprets links with  `master` as references to `main`, external sites like Binder do not (See #567).

🎯 **Goal**

To fix link rot.

💡 **Possible solutions**

Replace `master` with `main` for all relevant occurrences, throughout the repo.

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes.
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`.
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by prakashaditya369**
Fixes #570 

@JeS24

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -30,18 +30,19 @@ def setup(app):
 
 html_theme_options = {
     "logo": "logo_small.png",
+    "body_text_align": "left",
```
Comment (JeS24) [inline docs/source/conf.py:33]: Any particular reason to move these (`"body_text_align"` and `"donate_url"`?

File: README.rst [hunk 2]
```diff
@@ -32,18 +32,17 @@
...
 .. |codecov| image:: https://img.shields.io/codecov/c/github/einsteinpy/einsteinpy.svg?style=flat-square
-   :target: https://codecov.io/github/einsteinpy/einsteinpy?branch=master
-
-.. |appveyor| image:: https://img.shields.io/appveyor/ci/shreyasbapat/einsteinpy/master?logo=appveyor&style=flat-square
-   :target: https://ci.appveyor.com/project/shreyasbapat/einsteinpy/branch/master
+   :target: https://codecov.io/github/einsteinpy/einsteinpy?branch=main
 
+.. |appveyor| image:: https://img.shields.io/appveyor/ci/shreyasbapat/einsteinpy/main?logo=appveyor&style=flat-square
+   :target: https://ci.appveyor.com/project/shreyasbapat/einsteinpy/branch/main
 .. |orcid-shreyas| image:: https://img.shields.io/badge/id-0000--0002--0870--4665-a6ce39.svg
```
Comment (JeS24) [inline README.rst:46]: Add a blank line here (L46).

File: appveyor.yml [hunk 3]
```diff
@@ -10,7 +10,7 @@ environment:
 
 branches:
   only:
-    - master
+    - main
```
Comment (JeS24) [inline appveyor.yml:13]: Let's keep this change for a separate PR. This one's only for correcting link rot in documentation.

**Review by JeS24 — COMMENTED** [review]
Some comments.

File: docs/source/conf.py (hunk 1 above)
Comment (prakashaditya369) [reply docs/source/conf.py:33]: No, I think that changed while I was trying to work with badge_branch, I will reset this change.

**Comment by JeS24** [comment]
@prakashaditya369 Thanks for the PR. Merging now.

**Comment by prakashaditya369** [comment]
@JeS24 Thanks.
